src/popup/components/Modal.vue

```diff
@@ -41,7 +43,7 @@
           <BtnClose
             v-if="hasCloseButton"
             data-cy="btn-close"
-            @click="$emit('close')"
+            @click.capture="handleClose"
```

Comment:
Instead of using `capture` I suggest to update the plain button by removing the inner click event handler.